lib/db.js

```diff
@@ -278,9 +264,9 @@ DB.prototype.blockHandler = function(block, add, callback) {
   }
 
   async.eachSeries(
-    this.modules,
-    function(module, next) {
-      module['blockHandler'].call(module, block, add, function(err, ops) {
+    this.node.modules,
```

Comment:
I checked the behavior of `eachSeries` and it will iterate in the same way as an Array.